src/style.css

```diff
@@ -1,3 +1,59 @@
 body {
   margin: 0;
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+}
+
+.header {
+  text-transform: uppercase;
+  background-color: #fff;
+  padding: 10px 50px;
```

Comment:
the clickable area of the link should be the full height of the header, so 10px is not needed here

Reply:
Hello, it`s not like clickable area, it is like space between head of page and our header and the same 50px like space between left and right border to our header